app/assets/javascripts/application.js

```diff
@@ -23,4 +23,6 @@
 
 $(function() {
   $('.datepicker').datepicker();
+  setup_all_easy_submit();
+  setup_all_markdown_writing_preview();
```

Comment:
Identifier 'setup_all_markdown_writing_preview' is not in camel case.<br>'setup_all_markdown_writing_preview' is not defined.